frontend/src/main/utils/users.js

```diff
@@ -2,16 +2,60 @@ import { useQuery } from "react-query";
 import axios from "axios";
 
 export function useUsers() {
-  return useQuery("users", async () => {
-    const uri = "/api/admin/users";
-    try {
-      const response = await axios.get(uri);      
-      return response.data ;
-    } catch (e) {
-      console.error(`Error getting data from ${uri}:`,e);
-      return [];
+  return useQuery(
+    "users",
```

Comment:
So the query key here is "users".  It's going to be important that we invalidate that key as soon as you suspend or restore a user so that the table is updated.

Reply:
Done!